src/components/footer/footer.js

```diff
@@ -0,0 +1,159 @@
+        justifyContent: "center",
+        flexDirection: "column",
+        alignItems: "center",
+      },
+    },
+    imageInFooter: {
+      height: "150px",
+      width: "99%",
+      objectFit: "cover",
+      border: "solid 7px",
+      borderColor: WHITE_SOLID,
+      borderBottom: "none",
+    },
+    footerLogoAndParagraphsContainer: {
+      height: "650px",
+      backgroundColor: FLINT,
+      width: "99%",
+      border: "solid 7px",
+      borderColor: WHITE_SOLID,
+
+      borderTop: "none",
+    },
+    sectionOneInFooterLogoAndParagraphsContainer: {
+      width: "100%",
+      height: "100%",
+      display: "flex",
+      flexDirection: "column",
+      justifyContent: "space-around",
+      alignItems: "center",
+    },
+    sectionTwoInFooterLogoAndParagraphsContainer: {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "center",
+      marginTop: "60px",
+    },
+    logoInFooter: {
+      width: "150px",
+      height: "150px",
+      marginBottom: "10px",
+    },
+    paragraphsUnderLogo: {
+      fontFamily: "'Century Gothic', sans-serif",
+    },
+    emailLinkInSectionTwo: {
+      color: ZEUS,
+      textDecorationColor: ZEUS,
+    },
+    accountsLinksContainer: {
+      letterSpacing: "5px",
+      fontWeight: "bold",
+      fontFamily: "'Century Gothic', sans-serif",
+      marginBottom: "40px",
+    },
+    accountsLinks: {
+      color: BLACK_SOLID,
+      textDecoration: "none",
+      marginLeft: "10px",
+    },
+  },
+});
+
+export default function Footer() {
+  return (
+    <ThemeProvider theme={theme}>
+      <Box style={{ width: "100%" }} sx={theme.custom.footerContainer.sx}>
+        <img
+          alt="img-in-footer"
+          src="/carousel/pic1.jpg"
+          style={theme.custom.imageInFooter}
+        />
+        <Box style={theme.custom.footerLogoAndParagraphsContainer}>
+          <Box
+            style={theme.custom.sectionOneInFooterLogoAndParagraphsContainer}
+          >
+            <Box
+              style={theme.custom.sectionTwoInFooterLogoAndParagraphsContainer}
+            >
+              <img
+                src={logo}
+                alt="logo-in-footer"
+                style={theme.custom.logoInFooter}
+              />
+              <Typography
+                style={theme.custom.paragraphsUnderLogo}
+                variant="h5"
+                gutterBottom
+              >
+                GHAITH KONBAZ
+              </Typography>
+              <Typography
+                style={theme.custom.paragraphsUnderLogo}
+                variant="h6"
+                gutterBottom
+              >
+                DESIGHN
+              </Typography>
+            </Box>
+
+            <Typography
+              style={theme.custom.paragraphsUnderLogo}
+              variant="h6"
+              gutterBottom
+            >
+              FOR MORE INFORMATION, EMAIL{" "}
+              <Link style={theme.custom.emailLinkInSectionTwo} href="#">
```

Comment:
href="#" for now? 
are you tackling emailing in another PR?

Reply:
Yes I am, I will remove href for now.